pkg/descheduler/evictions/evictions.go

```diff
@@ -113,8 +116,13 @@ type EvictOptions struct {
 // EvictPod evicts a pod while exercising eviction limits.
 // Returns true when the pod is evicted on the server side.
 func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptions) bool {
+	attributes := trace.WithAttributes(attribute.String("podName", pod.Name), attribute.String("podNamespace", pod.Namespace), attribute.String("reason", opts.Reason))
```

Comment:
this doesn't need to be a variable, you can just pass this `trace.WithAttributes` in the `Tracer().Start(...trace.withAttributes)` below

Reply:
Turned this into a variable to avoid having to keep the lengthy lines having multiple `WithAttributes`, Will move that inline